sockio_darwin.go

```diff
@@ -0,0 +1,10 @@
+// +build darwin amd64
```

Comment:
Can you add copyright headers to this file (same for the linux file as well)

Reply:
Done.